docs/gestor_tareas.md

```diff
@@ -0,0 +1,6 @@
+# Elección del gestor de tareas que se usará
+A la hora de escoger el gestor de tareas, he barajado diferentes posibilidades:  
```

Comment:
Lo que quiero que me contéis son las posibilidades que *hay*, no las que habéis barajado. He comentado uno más en clase, que no estás incluyendo aquí.

Sería conveniente también que añadieras algún criterio por el que has usado uno u otro; por ejemplo, siendo bun útil para las dos cosas, cabría pensar que es una buena elección.